src/test/java/com/github/rholder/retry/RetryerBuilderTest.java

```diff
@@ -140,12 +143,8 @@ public void testRetryIfException() throws ExecutionException, RetryException {
                 .build();
         try {
             retryer.call(callable);
-            fail("RetryException expected");
-        } catch (RetryException e) {
-            assertEquals(3, e.getNumberOfFailedAttempts());
-            assertTrue(e.getLastFailedAttempt().hasException());
-            assertTrue(e.getLastFailedAttempt().getExceptionCause() instanceof IOException);
-            assertTrue(e.getCause() instanceof IOException);
+            fail("Exception expected");
+        } catch (RetryException ignored) {
```

Comment:
In case this floats your boat, check out https://github.com/joel-costigliola/assertj-core.  I specifically like [catchThrowable](https://github.com/joel-costigliola/assertj-core/blob/master/src/main/java/org/assertj/core/api/Assertions.java#L1197).